framework/src/media/InputHandler.h

```diff
@@ -45,7 +45,7 @@ class InputHandler : public StreamHandler
 	InputHandler();
 	void setInputDataSource(std::shared_ptr<InputDataSource> source);
 	bool doStandBy();
-
+	virtual bool close() override;
```

Comment:
We don't have to use virtual since we are already using override

Reply:
:o  Is it wrong or unnecessary?

Reply:
I got answer from cppreference: 
`In a member function declaration or definition, override ensures that the function is virtual and is overriding a virtual function from a base class. The program is ill-formed (a compile-time error is generated) if this is not true.`
I removed unnecessary `virtual` here.
Thanks for your review.